src/components/Disclosuers/Tabs/tabs.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+import React, { forwardRef } from "react";
+import { TabsProps } from "./tabs.type";
+import { Text } from "@texts/Text/Text";
+import styled from "styled-components";
+import { cx } from "@utils/cx";
+import { LMAsset } from "@utils/LumiaAssetManager";
+import { color as globalColor } from "@shared/styles";
+import { spacing, strokes } from "@shared/styles";
+import "./tabs.styles.css";
+
+const TabsComponent: React.ForwardRefRenderFunction<
+  HTMLAnchorElement,
+  TabsProps
+> = (
+  {
+    direction = "horizontal",
+    selectType = "select-100",
+    label,
+    icon,
+    selected = true,
+    color,
+    iconSize = 1,
+    ...props
+  },
+  ref
+) => {
+  const TabsContainer = styled.a`
```

Comment:
Keep the anchor tag as a container. Use the Flex component to use all the directional changes. as a children. As in future the anchor tag is gonna be replace by our own Link component.